dir.props

```diff
@@ -380,6 +383,8 @@
     <HostFxrDebPkgName>$(HostFxrDebPkgName.ToLower())</HostFxrDebPkgName>
     <SharedFxDebPkgName>$(DotnetRuntimeString)$(SharedFrameworkNugetVersion)</SharedFxDebPkgName>
     <SharedFxDebPkgName>$(SharedFxDebPkgName.ToLower())</SharedFxDebPkgName>
+    <RuntimeMetaDebPkgName>$(DotnetRuntimeMetaPackageString)$(HostResolverVersion)</RuntimeMetaDebPkgName>
```

Comment:
It is incorrect to use `$(HostResolverVersion)` version here.  It should be `$(ProductVersion)`